cwms_radar_api/src/main/java/cwms/radar/ApiServlet.java

```diff
@@ -213,10 +213,10 @@ protected void configureRoutes()
         get("/", ctx -> ctx.result("Welcome to the CWMS REST API").contentType(Formats.PLAIN));
         radarCrud("/location/category/{category-id}", new LocationCategoryController(metrics), requiredRoles);
         radarCrud("/location/group/{group-id}", new LocationGroupController(metrics), requiredRoles);
-        radarCrud("/locations/{location_code}", new LocationController(metrics), requiredRoles);
+        radarCrud("/locations/{location-id}", new LocationController(metrics), requiredRoles);
```

Comment:
Seems like it would be better to keeps these as `-name` ,or even just `name`.

Since `id` is traditionally used for a numeric value seems like it would reduce confusion.